client/src/pages/LoginPage.js

```diff
@@ -33,10 +67,15 @@ const LoginPage = () => {
                 placeholder="password"
                 type="password"
                 className="block w-full px-4 py-2 mt-2 bg-white border rounded-md focus:border-blue-400 focus:ring-blue-300 focus:outline-none focus:ring focus:ring-opacity-40"
+                onChange={(e) => [setPassword(e.target.value), setErrMsg(false)]}
               />
+              <ErrMsg error={errMsg} />
             </div>
             <div className="mt-16">
-              <button className="w-full px-4 py-2 tracking-wide text-white transition-colors duration-200 transform bg-regal-blue rounded-md hover:bg-blue-900 focus:outline-none focus:bg-blue-900">
+              <button
```

Comment:
- [ ] Add style on button to make it clear that this is disabled if the input fields are empty or utilize the validation that we set in BE. Either way is fine for me but would depend on client.